### PR TITLE
add app name as label for AppRevision

### PR DESCRIPTION
**Pull request by xiqiu618**
Fixes: #1476

### CODE REVIEW

**Review by wonderflow — COMMENTED** [review]
good job, can you add a test for it? You can just check it in some existing tests.

**Review by wonderflow — APPROVED** [review]
LGTM after CI passed

**Comment by wonderflow** [comment]
@xiqiu618 It seems the newly added test case has something wrong. Please fix it.

File: pkg/controller/core.oam.dev/v1alpha2/application/apply_test.go [hunk 1]
```diff
@@ -163,4 +166,19 @@ var _ = Describe("Test Application apply", func() {
 		Expect(strings.Compare(revision, preRevision) > 0).Should(BeTrue())
 	})
 
+	It("Test update or create app revision", func() {
+		ctx := context.TODO()
+		By("[TEST] Creating a app revision the first time")
+		apprev := &v1beta1.ApplicationRevision{}
+		apps := &v1beta1.ApplicationList{}
+		err := k8sClient.List(ctx, apps)
+		appRevName := apps.Items[0].Status.LatestRevision.Name
```
Comment (wonderflow) [inline pkg/controller/core.oam.dev/v1alpha2/application/apply_test.go:175]: the CI paniced maybe because of here

Comment (xiqiu618) [reply pkg/controller/core.oam.dev/v1alpha2/application/apply_test.go]: thanks, has been fixed

**Review by zzxwill — APPROVED** [review]
/lgtm

**Comment by wonderflow** [comment]
good job

**Pull request merged**